### ISSUE TITLE
Documentation Confusion

### ISSUE THREAD

**Issue by CubicLog** [public-awesome/constellations-book#6]
I'm writing this issue because I have no idea where else to ask for help. I'm trying to use the SALES event to get a list of recently purchased NFTs. I really don't understand NFTs and blockchains in general in much depth.

Looking through the documentation, I'm unsure about what each field is and what they are used for exactly. The documentation doesn't really explain it. The information I'm looking for is the NFT title, and image url for each sale. I'm unsure what field contains info about the NFT. All I could find was the buyer and seller addresses. I tried using the name for the buyer and selling addresses, but only some seem to work. Are you able to help or maybe add more information to the docs?

Again, I don't really know much about NFTs and that kind of stuff, so I'm sorry if I'm asking really dumb questions. I'm just another developer dragging my head across my desk. ¯\_(ツ)_/¯

Edit: I finally figured it out, but my suggestion for adding more descriptions to the docs still stands

**Comment by penso** [comment]
@CubicLog found your way? Let me know what you did and I'll add it to the documentation.

**Comment by CubicLog** [comment]
@penso Yes, I figured it out. I eventually realized that the NAME query could be used to get information about users including their display name using their address. I also came to the realization that each NFT belonged to a collection and that I could use the TOKEN query to get the name and image URL of the desired NFT using the collection address and tokenId from the SALES event. I think part of the confusion came from me not really understanding how NFTs work, but I do think the docs could provide more info about what each attribute is and what it can be used for.
